### ISSUE TITLE
Way to deploy GCE instance with predefined disk size/type

### ISSUE THREAD

**Issue by mrh666** [saltstack/salt#17097]
Hi there,

What is really will be helpful - it's to have functionality to deploy GCE instance with predefined disk size and to choose a type persistent/SSD.

Thank you.

**Comment by rallytime** [comment]
@mrh666 Apologies that we hadn't made it back around to this conversation. There was indeed a bug with libcloud 0.15.1 that made salt-cloud not work with it. However, now that newer versions of libcloud are out, GCE should be working with libcloud again.

I just tested this the other day and it is working well, as I noted [here](https://github.com/saltstack/salt/issues/19236#issuecomment-142694120). Apache-libcloud version 0.18.0 is out and working well with the GCE driver and Salt supports the `ex_disk_type: pd-ssd` starting with 2014.7.0.

I added documentation for this functionality in a PR yesterday, seen [here](https://github.com/saltstack/salt/pull/27345/files#diff-0d1d6f19977b0413cbac1a26a4f5cad2R254).

Given this information, is it alright if we close this bug?
